### ISSUE TITLE
Android Support

### ISSUE THREAD

**Issue by adarshjamuaar** [rapyuta-robotics/rclUE#137]
Is there any way to include this plugin with android build? Do I need to recompile ROS libraries?
Can someone please give me a step-wise process to make it compatible? 
Thanks!

**Comment by yuokamoto** [comment]
Hello,

I think yes, if you can build ros for android and put into [ThirdParty](https://github.com/rapyuta-robotics/rclUE/tree/devel/ThirdParty/ros), it should work but I haven't tried yet.

**Comment by adarshjamuaar** [comment]
@yuokamoto thanks for your reply! is there any guide to build ros for android that I can follow?

**Comment by yuokamoto** [comment]
I don't have experience on android development, but I found this https://github.com/ros2-java/ros2_java by google

**Comment by adarshjamuaar** [comment]
ah i see. kinda hard to follow :/

**Comment by adarshjamuaar** [comment]
hi any other help?

**Comment by yuokamoto** [comment]
Sorry, I don't have Android device and can't test it quickly...
